content/styles/03-components/presentation.css

```diff
@@ -55,12 +58,22 @@
     display: flex;
     align-items: center;
     justify-content: flex-end;
-    height: 35rem;
+    height: 32rem;
     width: 37.5rem;
     overflow: hidden;
 
     img {
       width: 100%;
     }
   }
+
+  .blog-preview {
+    display: flex;
+    flex-wrap: wrap;
+
+    .article-list {
```

Comment:
- I can't find a class `article-list` within a `.blog-preview` block.
- Also this could telescope the style with the same name from home.css, right?
- Why not reusing `preview-list`?

Reply:
Thanks for the catch! Adressed with b2f8501eb1e605b66ea5affc16dd5b7a0d098a54